R/modified_update.R

```diff
@@ -463,13 +515,26 @@ mu_impact_metadata <- function(meta, touchstone_use) {
 ##'
 ##' @param d Data: use impact_rate_tot (method 2)
 ##'
+##' @param method two options: method1 and method2
 ##' @export
-mu_scale <- function(name, d) {
-  ## This chunck becomes simpler, since only method 2 is used throughout.
-  rate_tot_old <- d[[paste0(name, "_rate_tot")]]
-  fvps_new <- d$fvps_new
-
-  ret <- fvps_new * rate_tot_old
+mu_scale <- function(name, d, method = "method2") {
+  ## This chunck becomes simpler, since only method 2 is used throughout. - Not any more, we need method 1 now.
```

Comment:
can you rewrite this comment to describe only the new code?

Reply:
Done.